### DISCUSSION TITLE
Option to disable spotify connect

### DISCUSSION THREAD

**Discussion by TanbirKang** [SpotX-Official/SpotX#400] [Ideas]
Would be a greatly appreciated feature considering spotify won't add it themselves.

**Comment by amd64fox** [comment]
why disable it?

**Reply by TanbirKang** [reply]
just really annoying when the playlist i was playing on my phone is automatically playing on my desktop when i switch. I listen to really different music when sitting down relaxing compared to when im out. Not a big deal but would just be a nice QOL feature. Ive seen quite a few requests for it on the official spotify forums and subreddit too.

**Reply by amd64fox** [reply]
just disable the Spotify desktop client when you're not using it, it's pretty weird to break Spotify Connect for that reason.
